guardian/support-frontend
# feat: enable link payment (0% AB test)

## What are you doing in this PR?

Adding Link payment to the generic checkout behind a 0% AB test.



[**Trello Card**](https://trello.com/c/aHoB0Ihf/1104-enabling-link-with-the-stripe-express-checkout-element)

## Why are you doing this?

We want to first test if this payment works in production. Secondly we want to launch an AB test to see if it improves CVR.



## How to test

Go to `/uk/checkout?product=Contribution&ratePlan=Monthly&contribution=3#ab-linkExpressCheckout=variant`. See Link payment button. Click it and complete the purchase.









## Screenshots
![image](https://github.com/user-attachments/assets/294e1de8-e3f9-4a2f-ada8-c3281e3e1c98)
